Makefile

```diff
@@ -62,19 +62,27 @@ work/imagenet/parsed_bb.csv: work/.imagenet_training_bbox_sentinel \
 	python motion_tracker/data_setup/parse_imagenet_bb.py data/imagenet/bb/ \
 		work/imagenet/parsed_bb.csv
 
+# Run after images are downloaded, to filter out those images 
```

Comment:
Are you just referencing the comment, or what's the issue with this? If it's the comment, happy to remove that. 

We can't have this run until imagenet images are downloaded and we know what failed, so it's dependent on that. But,I don't think that makes it a special case.